packages/page-files/src/CrustFiles.tsx

```diff
@@ -43,7 +43,7 @@ const shortStr = (name: string, count = 6): string => {
 };
 
 function createUrl (f: SaveFile) {
-  const endpoint = f.UpEndpoint || 'https://ipfs.io';
+  const endpoint = 'https://cf-ipfs.com';
```

Comment:
Why do we get rid of `f.UpEdnpoint`?

Reply:
Because currently UpEndpoint is only used for uploading files and cannot be used to access files.